Package.swift

```diff
@@ -1,22 +1,33 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.9
```

Comment:
Is this the lowest we can go?

Reply:
Sorry, this was just my feelings. 
The actual minimum Swift version is 5.8." (by swift-nio)